### PR TITLE
Bananas now have a similar chance to the Sussy firelocks to become Banana from Shovelware Studios Brain Game (as a treat)

### PR DESCRIPTION
**Pull request by Jolly-66**
## About The Pull Request
All in the title.
Its nothing more than a goofy little gag.

## Why It's Good For The Game
TLDR;
I'm clinically insane and want to see a funny not-dancing banana from Shovelware Studios jumpscare every botanist in existence when they harvest bananas.

Be lucky this isn't a minion or Mel or Gru. Actually we'd get in trouble with copyright with that wouldn't we?
Anyway, I don't know what I'm doing with my life anymore.

Picture because I know someone wants to know.
image

<details>
<summary>Not TLDR</summary>

It's a boring Thursday evening. I'm minding my business, had a good session of Barotramua with some friends. Just living life.

I get a notifcation from Discord server I'm in,
image

Y'know, I'm all for a good joke. Honestly. I originally thought to myself, "Oh God oh Jesus oh Fuck do not let the banana man near SS13". Yeah, that's what I thought.

image

This is where I went and asked Onule if they could whip up and sprite and that's where we are now.

There is now a potential Dancing Banana, from Shovelware Studios Brain Game, being added to Space Station 13.

What the fuck am I doing with my life?

</details>

## Changelog

:cl: Jolly, sprite by Onule
add: Jolly: Bananas have a 0.004% chance (much like the sussy firelocks, if you're on TG at least) to become a "special Banana".
imageadd: Onule: Added the "special Banana" sprite.
/:cl:

### CODE REVIEW

**Comment by Mothblocks** [comment]
the dancing banana is from peanut butter jelly time you gdod damn zoomer

**Comment by Mothblocks** [comment]
i think it should be a crossbreed between peanuts and bananas for that reason

**Comment by Mothblocks** [comment]
actually wait i remembered i hate when people add these direct blatant references especially on something extremely recent

**Comment by AverageUser67** [comment]
tgstation evil arc

**Comment by Mothblocks** [comment]
I'm being told they turned the peanut butter jelly time dancing banana into an NFT https://foundation.app/@PBJTIME/foundation/19082

**Comment by AverageUser67** [comment]
what the actual triple sweet and sour sauced deepfried fuck
internet evil arc

**Comment by LSommers** [comment]
You'll keep the shitass Pickle Rick reference ingame that looks like ass, is unwanted, and is a meme beaten to death, but you won't add the funny banana.

SOCIETY.

**Comment by optimumtact** [comment]
>You'll keep the shitass Pickle Rick reference ingame that looks like ass, is unwanted, and is a meme beaten to death, but you won't add the funny banana.

off topic for this PR but its' only still in teh game because some people can't behave on github, once the timer expires anyone who feels like they want to can remove it with a suitable replacement, hell this thing would even qualify.

**Comment by DAKKA-WAAAGH** [comment]
Maintainers, re-open this and merge it. I want to sacrifice 13 of these in the chapel to have an online admin play the song on repeat in order to torture the crew.

**Comment by Mothblocks** [comment]
if an online admin is willing to play the song they're probably willing to just put in the dmi too

**Comment by Sealed101** [comment]
we were this close to greatness
